### PR TITLE
Fix nested closures

### PR DESCRIPTION
**Pull request by uujava**
fixes NPE issue for closures with more than two level nesting

### CODE REVIEW

**Comment by felixvf** [comment]
Does this also fix https://github.com/mirah/mirah/issues/388?

File: src/org/mirah/typer/closures/binidng_adjuster.mirah [hunk 1]
```diff
@@ -0,0 +1,285 @@
+# Copyright (c) 2012-2015 The Mirah project authors. All Rights Reserved.
```
Comment (felixvf) [inline src/org/mirah/typer/closures/binidng_adjuster.mirah]: I guess you wanted to call the file ```binding_adjuster.mirah```.

Comment (uujava) [inline src/org/mirah/typer/closures/binidng_adjuster.mirah]: yep. should be renamed.

Comment (baroquebobcat) [inline src/org/mirah/typer/closures/binidng_adjuster.mirah]: I'll get it after the merge.

**Pull request merged**